### PR TITLE
connectors and optimize sm docs review

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Relates to https://github.com/camunda/developer-experience/issues/63.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: optimize/self-managed/optimize-deployment/version-policy.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 
-## Version Policy
-
 Camunda Optimize versions are denoted as X.Y.Z as well as by an optional [pre-release](https://semver.org/spec/v2.0.0.html#spec-item-9) version being either '-alpha-[0-9]' or '-preview-[0-9]'. X is the [major version](https://semver.org/spec/v2.0.0.html#spec-item-4), Y is the [minor version](https://semver.org/spec/v2.0.0.html#spec-item-7), Z is the [patch version](https://semver.org/spec/v2.0.0.html#spec-item-6) as defined by the [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html) specification.
 
-## Release Cadence
+## Release cadence
 
-Camunda Optimize has a new release every month. Every six months, a new minor/major is published, while all other months an alpha and/or patch release are released. The release dates are aligned with the rest of the [Camunda 8 Platform]($docs$/reference/release-policy).
+Camunda Optimize has a new release every month. Every six months, a new minor/major is published, while all other months an alpha and/or patch release are released. The release dates are aligned with the rest of [Camunda 8]($docs$/reference/release-policy).
```
Comment (RomanJRW) [inline optimize/self-managed/optimize-deployment/version-policy.md:11]: Maybe we should rephrase this to keep it a bit more general given that 8.4.0 doesn't fall into this cadence.

Maybe it would be enough to simply link to the C8 page without the description?

File: optimize/self-managed/optimize-deployment/version-policy.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
-## Pre-Release Versions
+## Pre-release versions
 
 There are two types of [pre-release](https://semver.org/spec/v2.0.0.html#spec-item-9) versions of Camunda Optimize that are published.
```
Comment (RomanJRW) [inline optimize/self-managed/optimize-deployment/version-policy.md:15]: I suspect this was because in the past we had preview releases too (although in practice only ever released one). We can probably just remove this whole section, alphas are the only pre-releases we currently do

**Review by RomanJRW — COMMENTED** [review]
Some minor comments from me 👍

**Review by akeller — APPROVED** [review]
I did not run these changes locally, but they look good to me. I would recommend merging ASAP and iterating on these (if necessary) so we can minimize the merge conflicts ahead of the minor release.
